graph/encoding/dot/encode.go

```diff
@@ -40,6 +40,10 @@ type Attributers interface {
 // connection ports for their end points. The returned port corresponds
 // to the the DOT node port to be used by the edge, compass corresponds
 // to DOT compass point to which the edge will be aimed.
+//
+// When encoded, the From() and To() sides of the graph.Edge will receive the
```

Comment:
I'd place this as separate comments on each of `FromPort` and `ToPort` rather than here.

```
type Porter interface {
    // FromPort returns the port and compass for the
    // From node of a graph.Edge.
    FromPort() (port, compass string)

    // ToPort returns the port and compass for the
    // To node of a graph.Edge.
    ToPort() (port, compass string)
}
```